django_topics/models/featured_topic.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from datetime import datetime
+from django.utils.timezone import now
+from django_topics.models import Topic
+
+
+class FeaturedTopicManager(models.Manager):
+
+    def get_featured_topic(self, lang: str, date: datetime = None) -> 'TopicOfTheDay':
+        """
+        Return featured topic for given date or closest date that is less than or equal to given date
+        @param lang: language code, "en" or "he"
+        @param date: datetime object
+        @return:
+        """
+        date = date or now().date()
+        return (
+            self.filter(start_date__lte=date, lang=lang)
+            .order_by('-start_date')
+            .first()
+        )
+
+
+class TopicOfTheDay(models.Model):
+    topic = models.ForeignKey(
+        Topic,
+        on_delete=models.CASCADE,
+        related_name='topic_of_the_day'
+    )
+    start_date = models.DateField()
+    lang = models.CharField(max_length=2, choices=[('en', 'English'), ('he', 'Hebrew')])
+    objects = FeaturedTopicManager()
+
+    class Meta:
+        unique_together = ('topic', 'start_date')
+        verbose_name = "Landing Page - Featured Topic"
+        verbose_name_plural = "Landing Page - Featured Topic"
+
+    def __str__(self):
+        return f"{self.topic.slug} ({self.start_date})"
+
+
+class FeaturedTopicEnglish(TopicOfTheDay):
```

Comment:
I renamed every variable and class name to be based on the new name of the feature "Featured Topic" EXCEPT for the main django model which I kept as "TopicOfTheDay". This is unfortunate but this is why:
- TopicOfTheDay has already been merged into master
- Therefore, all cauldrons (or most) are dependent on this model
- TopicOfTheDay directly corresponds to a table in Postgres by the same name
- If we rename the model, it will rename the table which will break all cauldrons
- I want to wait to do this until my code is in master.